internal/cephfs/store/volumegroup.go

```diff
@@ -0,0 +1,310 @@
+/*
+Copyright 2024 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package store
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/ceph/ceph-csi/internal/cephfs/core"
+	cerrors "github.com/ceph/ceph-csi/internal/cephfs/errors"
+	fsutil "github.com/ceph/ceph-csi/internal/cephfs/util"
+	"github.com/ceph/ceph-csi/internal/util"
+	"github.com/ceph/ceph-csi/internal/util/log"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+)
+
+type VolumeGroupOptions struct {
+	VolumeOptions
+}
+
+// Connect a CephFS volume to the Ceph cluster.
+func (vo *VolumeGroupOptions) Connect(cr *util.Credentials) error {
```

Comment:
how is this different from `VolumeOptions.Connect()`? Same for other functions.

Reply:
Yes this will be same as `VolumeOptions.Connect()` should have removed when we embedded `VolumeOptions` to volumeGroupOptions`. removed both connect and Destroy for now.